kubernetes/yamls/development/api-server/api-deployment.yml

```diff
@@ -18,13 +18,8 @@ spec:
           httpGet:
             path: /api/status.json
             port: 80
-          initialDelaySeconds: 30
-          timeoutSeconds: 3
-        readinessProbe:
-          httpGet:
-            path: /api/status.json
-            port: 80
-          initialDelaySeconds: 30
+          initialDelaySeconds: 90
+          periodSeconds: 300
           timeoutSeconds: 3
```

Comment:
1) I didn't say ElasticSearch is working fine. I showed you logs that says it isn't working fine :sweat_smile:
2) yes, persistant disk exists
3) Everything is as in the PR
```
# External elasticsearch:
# If elasticsearch_transport.enabled is set to true, then an external elasticsearch is used
# The external elasticsearch server must have one configuration change to enable the access of
# loklak: in elasticsearch/config/elasticsearch.yml must be added one line, containing
# cluster.name: loklakcluster
elasticsearch_transport.enabled = false
elasticsearch_transport.addresses = 127.0.0.1:9300
elasticsearch_transport.cluster.name = loklakcluster
elasticsearch_requiredClusterHealthStatus = YELLOW

# The following are elasticsearch properties:
# The prefix 'elasticseach.' is removed and handed over to the elasticsearch node
elasticsearch.cluster.name=index
elasticsearch.cluster.routing.allocation.disk.threshold_enabled=true
elasticsearch.cluster.routing.allocation.disk.watermark.low=4gb
elasticsearch.cluster.routing.allocation.disk.watermark.high=2gb
elasticsearch.http.enabled=true
elasticsearch.http.cors.enabled=true
elasticsearch.http.cors.allow-origin=/https?:\/\/localhost(:[0-9]+)?/
elasticsearch.http.port=9200
elasticsearch.transport.tcp.port=9300
elasticsearch.network.host=localhost
elasticsearch.node.data=true
elasticsearch.node.master=true
elasticsearch.index.number_of_shards=8
elasticsearch.index.number_of_replicas=0
elasticsearch.index.auto_expand_replicas=false
elasticsearch.index.refresh_interval=10s
elasticsearch.index.max_result_window=10000
elasticsearch.path.conf=conf
elasticsearch.path.data=data
elasticsearch.path.home=data
elasticsearch.elasticsearch_discovery_zen_ping_multicast_enabled = false
elasticsearch.indices.fielddata.cache.size=10%
elasticsearch.index.translog.flush_threshold_size=8mb
#elasticsearch.path.logs=data/log
```